test_util/provide_aws.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+import logging
+import os
+from abc import ABCMeta, abstractmethod
+
+import boto3
+import botocore
+import retrying
+
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+AWS_VERIFY = True
```

Comment:
I believe this is the default and we'd never want to set it differently. Do we need this as a parameter?